PostHog/PostHogSDK.swift

```diff
@@ -315,7 +372,12 @@ let maxRetryDelay = 30.0
         ))
     }
 
-    @objc public func alias(_ alias: String, properties: [String: Any]? = nil) {
+    @objc public func alias(_ alias: String) {
+        self.alias(alias, properties: nil)
+    }
+
+    @objc(aliasWithAlias:properties:)
+    public func alias(_ alias: String, properties: [String: Any]? = nil) {
```

Comment:
Similar to $identify, I'm not sure we actually want to expose the other properties.

Reply:
I agree to remove it from here, JS and JS-Lite don't have it, I probably just thought that it'd be useful.